conf/flink-conf.yaml

```diff
@@ -0,0 +1,177 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+#==============================================================================
+# Mesos
+#==============================================================================
+
+mesos.master: zk://zk-1.zk:2181/mesos
+mesos.failover-timeout: 60
+mesos.resourcemanager.tasks.container.image.type: mesos
```

Comment:
This config setting should be `mesos.resourcemanager.tasks.container.type` since `mesos` is the container type not the image type.  I realize this comment relates to the code, not to this commit really - sorry.